examples/allennlp/allennlp_jsonnet.py

```diff
@@ -0,0 +1,59 @@
+"""
+Optuna example that optimizes a classifier configuration for IMDB movie review dataset.
+This script is based on the example of allentune (https://github.com/allenai/allentune).
+
+In this example, we optimize the validation accuracy of
+sentiment classification using AllenNLP cli.
+Since it is too time-consuming to use the entire dataset, we here use a small subset of it.
+
+We have the following two ways to execute this example:
+
+(1) Execute this code directly.
+    $ python allennlp_cli.py
+
+
+(2) Execute through CLI.
+    $ STUDY_NAME=`optuna create-study --direction maximize --storage sqlite:///example.db`
+    $ optuna study optimize allennlp_cli.py objective --n-trials=100 --study $STUDY_NAME \
+      --storage sqlite:///example.db
+
+"""
+
+import os.path
+import shutil
+
+import optuna
+from optuna.integration import AllenNLPExecutor
+
+
+MODEL_DIR = "result"
+
+
+def objective(trial):
+
+    trial.suggest_uniform("DROPOUT", 0.0, 0.5)
+    trial.suggest_int("EMBEDDING_DIM", 20, 50)
+    trial.suggest_int("MAX_FILTER_SIZE", 3, 6)
+    trial.suggest_int("NUM_FILTERS", 16, 32)
+    trial.suggest_int("HIDDEN_SIZE", 16, 32)
+
+    config_path = "examples/allennlp/classifier.jsonnet"
```

Comment:
Oh, you're right.
I assume `allennlp_jsonnet.py` is executed from the root of this repository
since [an example test](https://github.com/optuna/optuna/blob/master/.github/workflows/examples.yml#L45-L53) seems to run scripts in `examples/`.
However, as you say, it is confusing for users to try this example.
If you have some good ideas, please let me know.

Reply:
I have stupid idea.

```python
os.path.join(os.path.dirname(os.path.abspath(__file__)), "classifier.jsonnet")
```

Reply:
Thank you for the good suggestion.
It's reasonable for me. fd1e567